cptoml.py

```diff
@@ -156,57 +167,51 @@ def _tablefind(buf, subtable) -> int:
 
     Returns -1 when not found.
     """
-    result = -1
-    tm = 0
-    q = True
-    while q:
-        tml = None
+
+    for index, line in enumerate(buf):
         try:
-            tml = _prepareline(buf[tm])
+            tml = _prepareline(line)
+
+            if tml == f"[{subtable}]":
+                return index
+
         except IndexError:
-            q = False
-        if q and tml.startswith("[") and (tml == f"[{subtable}]"):
-            result = tm
-            q = False
-        del tml
-        tm += 1
-        if not q:
             break
-    del buf, subtable, tm, q
-    return result
+
+    del buf, subtable
+    return -1
 
 
 def _getkeys(buf, start=0):
     """
     Get the keys off a table.
     """
-    res = list()
-    q = True
+    res = []
```

Comment:
Slower, uses more ram.

Reply:
At least on CPython, the constructor is slower than the literal. 

For the RAM usage, i dont see how these wouldn't match, you end up with an empty list in both cases (same class with same data)